### ISSUE TITLE
Build fails with Error: Cannot find module './lexer' <- NVM problem

### ISSUE THREAD

**Issue by mormy** [butterproject/butter-desktop#182]
Hi, 

I have had problems building butter ... Anyway, webchimaera.js fails with this error (and would fail before the compilation step if you leave out the har-validator npm package)... 

compiler output -> https://gist.github.com/anonymous/c9e0767467bf503d3e6f
```
-- Configuring incomplete, errors occurred!
See also "/Users/mormy/repos/js/butter-desktop/node_modules/webchimera.js/build/CMakeFiles/CMakeOutput.log".
ERR! OMG Process terminated: 1
Unhandled rejection Error: Process terminated: 1
    at ChildProcess.<anonymous> (/Users/mormy/repos/js/butter-desktop/node_modules/cmake-js/lib/es6/cMake.js:376:28)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```
And grunt throws this error, which is strange... Probably due to changes how coffee-script should have registered ... ```var coffee = require('coffee-script').register()``` for coffee-script > 1.7 ... 
```
module.js:339
    throw err;
    ^
Error: Cannot find module './lexer'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/mormy/repos/js/butter-desktop/node_modules/grunt/node_modules/coffee-script/lib/coffee-script/coffee-script.js:10:10)
    at Object.<anonymous> (/Users/mormy/repos/js/butter-desktop/node_modules/grunt/node_modules/coffee-script/lib/coffee-script/coffee-script.js:167:4)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
```

So basically, I traced the bug to nvm, so if you get these kinds of errors, culprit is nvm. Once I used the brewed node 5.0.0 it build OOB with the make_butter.sh script... 

It's not a bug per-se, but if you stumble upon this...

Try to remove your $HOME/node_modules and then run the script ... Or in the extreme case remove nvm... 

I'll test this out some more, but for now, I'm 99% positive that nvm f**ks up the build. 

OSX 10.11, MBP ... node 5.0.0, npm 3.3.9 ... installed via homebrew = working ... same node/npm installed via nvm, not working.
